components/nitro_utils/BUILD.gn

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
what is nitro and why should this be its own component instead of being part of p3a?

Reply:
addressed above